### ISSUE TITLE
Items disappear from inventory and hotbar on refill

### ISSUE THREAD

**Issue by Isaacw24** [jsnimda/Inventory-Profiles#29]
For example: using rockets to fly and i have an extra stack, upon refill it just makes the items disappear.
Please fix this as this is a great mod and my favourite to say that :) thanks!

**Comment by JohnyDL** [comment]
Yep me too, actually happens with a bunch of the people I'm on a server with if we're placing blocks and have 1 spare stack we get it put into our hand but like a ghost it's only in our hands once we place/use 1 item, or open our inventories to look, if we've got 2 stacks or more then 1 stack will move to our hand right away and the second stack will ghost (sometimes losing an item) and only reappear when we look in chests/shulkers/etc or pick up the item from what appeared to be an empty slot meaning that 1/2 the blocks in inventories can't be used without fiddling to some degree or another
